tests/handler/handler_invoke_api.cpp

```diff
@@ -30,18 +30,23 @@ using accessor_t =
                        cl::sycl::access::target::global_buffer>;
 
 struct single_task_functor {
-  single_task_functor(accessor_t acc) : acc(acc) {}
+  single_task_functor(accessor_t acc, size_t range) : acc(acc), range(range) {}
 
-  void operator()() { acc[0] = 10; }
+  void operator()() {
```

Comment:
```suggestion
  void operator()() const {
```

I would appreciate if we can make all call operators constant to ensure compatibility with SYCL-2020.

Reply:
Sure thing, done!